requirements-azure.txt

```diff
@@ -1,3 +1,3 @@
 azure-datalake-store >= 0.0.30
-azure-storage-blob
+azure-storage-blob ~= 2.1
```

Comment:
Thanks for identifying this. I'll look into this fix for the upstream changes independently as well/